gateway/README.md

```diff
@@ -108,6 +108,8 @@ The size of the machine that hosts the gateway depends on the amount of data tha
 
 ### INSTALLATION
 
+Note: If you are an APM customer, install the Smart Gateway instead.
```

Comment:
Include link to https://docs.signalfx.com/en/latest/apm/apm-deployment/smart-gateway.html#install-and-configure-the-smart-gateway

Reply:
Added link to "Smart Gateway"